Cargo.toml

```diff
@@ -78,7 +78,7 @@ license = "Apache-2.0"
 repository = "https://github.com/risingwavelabs/risingwave"
 
 [workspace.dependencies]
-foyer = { version = "0.11.3", features = ["mtrace", "nightly"] }
+foyer = { git = "https://github.com/foyer-rs/foyer", rev = "ab65b31", features = ["mtrace", "nightly"] }
```

Comment:
Revert this change after #18693?

Reply:
Yes, will merge main after #18693 merged.